cli/cmd/lib_cli_config.go

```diff
@@ -144,6 +147,35 @@ func cliEnvPromptValidation(defaults *CLIEnvConfig) *cr.PromptValidation {
 	}
 }
 
+func validateOperatorEndpoint(endpoint string) (string, error) {
+	url, err := cr.GetURLValidator(false, false)(endpoint)
+	if err != nil {
+		return "", err
+	}
+
+	req, err := http.NewRequest("GET", url+"/isthiscortex", nil)
+	if err != nil {
+		return "", errors.Wrap(err, "verifying operator endpoint", url)
+	}
+	req.Header.Set("Content-Type", "application/json")
+	bodyBytes, err := _apiClient.MakeRequest(req)
```

Comment:
It may be misleading to use `_apiClient`, perhaps make a new client here or in the other file?